web/src/components/molecules/Common/ShadowCard/index.tsx

```diff
@@ -0,0 +1,17 @@
+import Card from "@marketplace/components/atoms/Card";
+import { styled } from "@marketplace/theme";
+
+export type Props = {
+  children: React.ReactElement;
+};
+
+const ShadowCard: React.FC<Props> = ({ children }) => {
+  return <CustomCard>{children}</CustomCard>;
+};
+
+const CustomCard = styled(Card)`
+  background-color: #fafafa;
```

Comment:
Same here. Need to make sure any background-color doesn't break UI in extension.

Reply:
Thanks for spotting this. Changed it to `transparent`